p2panda-js/src/wasm.test.ts

```diff
@@ -100,21 +100,21 @@ describe('WebAssembly interface', () => {
       );
 
       expect(() => fields.add('contact', 'relation', [TEST_HASH])).toThrow(
-        'Expected a hash value for field of type relation',
+        'Expected an operation id value for field of type relation',
       );
 
       expect(() => fields.add('contact', 'relation_list', TEST_HASH)).toThrow(
-        'Exptected an array of hashes for field of type relation list',
+        'Exptected an array of operation ids for field of type relation list',
       );
 
       expect(() => fields.add('contact', 'pinned_relation', TEST_HASH)).toThrow(
-        'Expected an array of hashes for field of type relation list',
+        'Expected an array of operation ids for field of type relation list',
```

Comment:
This shoulf be "...of type pinned relation" right? (not related to this PR, but I just spotted it...)

Reply:
good catch!

Reply:
If you're changing this you can also fix the old `Exptected` typo, hehe.